assets/js/components/keyboardShortcutsModal.js

```
@@ -0,0 +1,481 @@
+    }
+    
+    /**
+     * Handle global keyboard shortcuts
+     * @param {KeyboardEvent} e - Keyboard event
+     */
+    handleGlobalShortcuts(e) {
+        switch(e.key.toLowerCase()) {
+            case 't':
+                // Toggle theme
+                const themeToggle = document.getElementById(ELEMENT_IDS.THEME_TOGGLE);
+                if (themeToggle) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)